projects/maixpy_bit/config_defaults.mk

```diff
@@ -0,0 +1,155 @@
+# Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib)
```

Comment:
Can you start with this config file as a template and change it as needed?
https://github.com/selfcustody/MaixPy/blob/master/projects/maixpy_m5stickv/config_defaults.mk

I made a lot of changes to these config files recently and it doesn't look like those changes made it into this one